wiki/en/en-Compiling.md

```diff
@@ -11,7 +11,7 @@ permalink: "/wiki/Compiling"
 
 Please refer to the [client installation page](Installation-for-Linux) or these instructions if you wish to [compile a headless server](Server-Linux#running-a-headless-server)
 
-## MacOS
+## macOS
```

Comment:
Could be dangerous for anchor links (but I don't think so).

Reply:
In my fresh memory, there is not internal links trageting this and the others.

Reply:
Ok. So they should be fine. We can check it on the next version.